downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/GenerateGOAnnotationFile/CellularComponentAnnotationBuilder.java

```diff
@@ -0,0 +1,59 @@
+package org.reactome.release.downloadDirectory.GenerateGOAnnotationFile;
+
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+
+import java.util.*;
+
+public class CellularComponentAnnotationBuilder {
+
+    private static final List<String> speciesWithAlternateGOCompartment = new ArrayList<>(Arrays.asList("11676", "211044", "1491", "1392"));
+    private static final List<String> microbialSpeciesToExclude = new ArrayList<>(Arrays.asList("813", "562", "491", "90371", "1280", "5811"));
```

Comment:
It looks like this list occurs in this class and the MolecularFunctionAnnoationBuilder.  It should probably be consolidated to one place if it has to be updated.  It might be worth creating a constants class for this step since there are many different ones.  We could also take the opportunity to give each species and compartment on these lists a meaningful name so its clear what the numeric values are.